src/pages/non-console/index.jsx

```diff
@@ -13,6 +13,7 @@ import SideNavigation from '@cloudscape-design/components/side-navigation';
 import SpaceBetween from '@cloudscape-design/components/space-between';
 import Table from '@cloudscape-design/components/table';
 import TopNavigation from '@cloudscape-design/components/top-navigation';
+import { isVisualRefresh } from '../../common/apply-mode';
```

Comment:
Why do we need the `isVisualRefresh` in the OS version?

Reply:
Good question, it is in several places in the demos